testplugins/src/main/java/org/spongepowered/test/MainTestPlugin.java

```diff
@@ -0,0 +1,94 @@
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.test;
+
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.command.spec.CommandExecutor;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.event.Listener;
+import org.spongepowered.api.event.game.state.GameInitializationEvent;
+import org.spongepowered.api.plugin.Plugin;
+import org.spongepowered.api.plugin.PluginContainer;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.format.TextColors;
+
+import java.util.Collection;
+
+@Plugin(id = "main_test", name = "Main Test", version = "0.0.0", description = "main-test")
+public class MainTestPlugin {
+
+    @Listener
+    public void onInit(GameInitializationEvent event) {
+        Sponge.getCommandManager().register(this,
+                CommandSpec.builder()
+                        .arguments(GenericArguments.optional(GenericArguments.plugin(Text.of("plugin"))))
+                        .executor(((src, args) -> {
+                            Collection<PluginContainer> plugins = args.getAll(Text.of("plugin"));
+                            if (plugins.isEmpty()) {
+                                plugins = Sponge.getPluginManager().getPlugins();
+                            }
+                            for (PluginContainer pc : plugins) {
+                                pc.getInstance().ifPresent(plugin -> {
+                                    if (plugin instanceof LoadableModule) {
+                                        Sponge.getServer().getBroadcastChannel().send(Text.of(pc.getId(), " enabled"));
+                                        ((LoadableModule) plugin).enable(src);
+                                    } else {
+                                        Sponge.getServer().getBroadcastChannel().send(Text.of(pc.getId(), " is not a LoadableModule"));
+                                    }
+                                });
+                            }
+                            return CommandResult.success();
+                        }))
+                        .build(), "enableplugins");
```

Comment:
`ep` and `dp`: shorter alias, faster to type 
`test enable|disable <plugin1> <plugin2>`: cleaner

Thoughts ?

Reply:
im fine with dropping the short alias, if you can make `/test` without any arguments a quick toggle to enabled/disable

Reply:
I would need to track the status (enabled/disabled) of individual plugins then. That would become uglier than the alias IMO.

Reply:
I don't see why the test plugins suite couldn't have an enabled/disabled container, would be rather interesting to see as a centralized debugging spot to visualize what plugins are enabled in the event of some listeners being active or inactive. Granted, a majority of these plugins are for testing functionality, and/or bug fixes, so it's not common to have this sort of interaction being preformed. My two cents are to have a global container (specific to the test plugins of course) to keep track of that.